src/MixedModels.jl

```diff
@@ -59,6 +63,7 @@ export @formula,
     ReMat,
     SeqDiffCoding,
     SqrtLink,
+    Table,
```

Comment:
do we need to export this?

Reply:
I would like to.  See issue #662.  I have made a change in this PR so that the `tbl` field of a `MixedModelProfile` is `Table` and not a `Vector{NamedTuple}`.  This means that many of the calls to `Table` that I had in code, just to format that table nicely in the display. are no longer needed.  But I would still like to have the `Table` constructor available.